scripts/install-build-deps

```diff
@@ -137,4 +137,4 @@ if ! $install_epel ; then
 fi
 
 $TOP_SRCDIR/scripts/setup-node localhost \
-    ${dry_run} --verbose --local --skip-tags $skip_tags
+    ${dry_run} --verbose --local --skip-tags $skip_tags -v -v -v
```

Comment:
Why?

Reply:
More verbose to see the every command used in the provisioning.
Sometimes, error happens, without logs. It is verify difficult to know what goes wrong.

I can not use "-v -v -v" to the "scripts/install-build-deps", because currently it does not pass these to the "setup_node"